src/main/java/uk/gov/hmcts/reform/ccd/documentam/service/impl/DocumentManagementServiceImpl.java

```diff
@@ -549,7 +549,7 @@ private HttpHeaders getHttpHeaders() {
 
     private void catchException(HttpClientErrorException exception, String messageParam) {
```

Comment:
rename to describe what it does better `catchAndRethrowException` ?

Reply:
I agree with @sullivangareth that we could rename this method for clarity

Reply:
sure, that's unchanged line 😄 . 
There are few other methods like that in that class.. will rename all together